### ISSUE TITLE
Getting HTML error instead of JSON error when retrieving a non-existing job run ID

### ISSUE THREAD

**Issue by zhiyiliu** [MarquezProject/marquez#511]
If I use an ID not in UUID format, it would return JSON response.  Not sure if this is only limited to Retrieve, or if any other operations by UUID has this issue.

GET http://localhost:8080/api/v1/jobs/runs/1234
```
Response: Content-Type →application/json

{
    "code": 404,
    "message": "HTTP 404 Not Found"
}
```
But if I use a bogus UUID format ID, it would return HTML page.
GET http://localhost:8080/api/v1/jobs/runs/683b678f-52ce-452f-b3ad-efa76a680b56

```
Response: Content-Type →text/html;charset=iso-8859-1

<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
        <title>Error 404 Not Found</title>
    </head>
    <body>
        <h2>HTTP ERROR 404</h2>
        <p>Problem accessing /api/v1/jobs/runs/683b678f-52ce-452f-b3ad-efa76a680b56. Reason:

            <pre>    Not Found</pre>
        </p>
    </body>
</html>
```

**Comment by zhiyiliu** [comment]
Also found other behaviors for other job run operations, start/complete/fail/abort.  

PUT http://localhost:8080/api/v1/jobs/runs/1234/abort Will return 
```
Content-Type →text/plain
Could not parse 1234 into a UUID!
```

**Comment by wslulciuc** [comment]
@zhiyiliu, thanks for opening this issue! We have fixed this error (see #685) and will be part of release `0.9.0`

Using the example _invalid_ UUID you've provided above `1234`:

```bash
$ curl -X POST http://localhost:5000/api/v1/jobs/runs/1234/abort | jq .
```

You will now see the API error:

```json
{
  "code": 400,
  "message": "Run '1234' is invalid."
}
```
